samples/force-sensitive-resistor/force-sensitive-resistor.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
Suggestion: Create a variable for the default target framework for the samples. Search and replace on 100+ files each time we want to upgrade is a bit ugly.

Reply:
Good point, will do.